app.py

```diff
@@ -138,6 +150,13 @@ def get_needs_by_location():
 def ping():
     return "pong"
 
+@app.route("/api/v1/location", methods=["GET"])
+def getLocationFromString():
+    parameters = {'input': request.args.get('address'), 'inputtype': 'textquery', 'key': GOOGLE_API_KEY, 'fields': 'geometry'}
+    response = requests.get('https://maps.googleapis.com/maps/api/place/findplacefromtext/json', params=parameters)
+    if response.status_code == 200:
+        return json.dumps(response.json()), 200, {'Content-Type': 'application/json'}
```

Comment:
Shall we use jsonify instead of json.dumps as it consistent with rest of the API.

Reply:
Will push the changes. Give me some time.

Reply:
Replaced with jsonify. Please check now.